Alune.spec

```diff
@@ -0,0 +1,39 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
+a = Analysis(
+    ['main.py'],
+    pathex=[],
+    binaries=[],
+    datas=[('alune/images', 'alune/images')],
```

Comment:
Is this duplication necessary?

Reply:
Yes, at least as far as I understood it, it's (origin, target).